src/components/data-table-v2/_data-table-v2-expandable.scss

```diff
@@ -25,19 +34,34 @@
     }
 
     + tr[data-child-row] {
+      display: table-row;
       td {
         border-top: 0;
         padding-bottom: $spacing-xs;
+        max-height: 100%;
```

Comment:
Seems that change here is to stable code, is it correct? Just wanted to double-check if this is intentional.

Reply:
Yes, this was intentional. This was meant to be more of an update to stable data table so that is can also include motion on expansion. Because the js it was using just completely removed the child row from the DOM, it didn't include any motion, but since designers said ideally it should have motion, I updated the experimental version to use more css than js to be able to add transitions, and I thought I should update the stable version as well to add transitions.

Reply:
Thanks @jnm2377 for clarifying! We tend to avoid sudden visual changes in minor releases unless it's minor, esp. given consumers tend to auto-upgrade Carbon in their daily builds. Just wanted to see if it's in such category, while we may be in sync with our designers on this.

Reply:
As long as it's not breaking, I think adding motion to our components should be encouraged